topic_camera: "/dope/webcam_rgb_raw"
topic_publishing: "dope"
frame_id: "/dope"

# Comment any of these lines to prevent detection / pose estimation of that object
weights: {
    "003_cracker_box":"cracker_60.pth", 
    # "gelatin":"gelatin_60.pth",
    "010_potted_meat_can":"meat_20.pth", 
    "006_mustard_bottle":"mustard_60.pth", 
    "005_tomato_soup_can":"soup_60.pth", 
    # "sugar":"sugar_60.pth"
}

# Cuboid dimension in cm x,y,z
dimensions: {
    "003_cracker_box": [16.403600692749023,21.343700408935547,7.179999828338623],
    "gelatin": [8.918299674987793, 7.311500072479248, 2.9983000755310059],
    "010_potted_meat_can": [10.164673805236816,8.3542995452880859,5.7600898742675781],
    "006_mustard_bottle": [9.6024150848388672,19.130100250244141,5.824894905090332],
    "005_tomato_soup_can": [6.7659378051757813,10.185500144958496,6.771425724029541],
    "sugar": [9.267730712890625,17.625339508056641,4.5134143829345703],
}

draw_colors: {
    "003_cracker_box": [13, 255, 128],  # green
    "gelatin": [255, 255, 255],  # while
    "010_potted_meat_can": [0, 104, 255],  # blue
    "006_mustard_bottle": [217,12, 232],  # magenta
    "005_tomato_soup_can": [255, 101, 0],  # orange
    "sugar": [232, 222, 12],  # yellow
}

# Camera intrinsics (Logitech C920)
# camera_settings: {
#     "name": "logitech_c920",
#     "width": 640,
#     "height": 480,
#     "fx": 641.5,
#     "fy": 641.5,
#     "cx": 320.0,
#     "cy": 240.0
# }

# fetch 7
camera_settings: {
    "name": "fetch 7",
    "width": 640,
    "height": 480,
    "fx": 542.556,
    "fy": 542.556,
    "cx": 322.065,
    "cy": 222.214
}

# Config params for DOPE
thresh_angle: 0.5
thresh_map: 0.01
sigma: 3
thresh_points: 0.1
